src/pages/public_registration/LogPanel.tsx

```diff
@@ -204,8 +194,43 @@ export const LogPanel = ({ tickets, registration }: LogPanelProps) => {
   );
 };
 
+interface FilenamesProps {
+  filenames: string[];
+}
+
+const FilenamesList = ({ filenames }: FilenamesProps) => {
+  return (
+    <Box
+      component="ul"
+      sx={{
+        gridColumn: '1/3',
+        m: '0',
+        p: 'inherit',
+        color: 'black',
+      }}>
+      {filenames.map((filename, index) => (
+        <li key={index}>
```

Comment:
React er ikke så begeistret for å bruke `index` som `key`, da det kan føre til uforventede rerenders.
```suggestion
        <li key={`${filename}-${index}`}>
```

Reply:
Ah, will fix!